buku.py

```diff
@@ -1492,6 +1492,31 @@ def get_page_title(resp):
         return htmlparser.parsed_title
 
 
+def create_poolmanager(url):
```

Comment:
I was trying to be general, but it's alright. Would you have any objections for using ProxyManager in place of PoolManager? And should I add a new parameter or just adapt to the environment variables?